app/models/events.rb

```diff
@@ -0,0 +1,22 @@
+# Object representing relevant actions taken by a case  manager
+class Event
+  include Mongoid::Document
+  include Mongoid::Timestamps
+  include Mongoid::History::Trackable
```

Comment:
I think since these are disposable and immutable events, my inclination is to ditch historytrackable and userstamp here. Thoughts?